src/include.am

```diff
@@ -300,10 +305,15 @@ endif
 endif !BUILD_FIPS_CURRENT
 
 if !BUILD_FIPS_CURRENT
+src_libwolfssl_la_SOURCES += wolfcrypt/src/sha256.c
 if BUILD_ARMASM
 src_libwolfssl_la_SOURCES += wolfcrypt/src/port/arm/armv8-sha256.c
+if BUILD_ARMASM_INLINE
```

Comment:
same again, and if its not defined it will also build build the Asm

Is this correct?

Reply:
Yes, as above.